Bottom: cf8998b9f06f893b499b9478f40610fb02b1993a
Top:    26d22c27ce9f6ea665ee146c378b4269df197ab2
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2013-01-09 14:54:47 -0800

rspreload: Add support for existing UDP apps

Support existing UDP applications through the UDP library.

Signed-off-by: Sean Hefty <sean.hefty@intel.com>


---

diff --git a/src/preload.c b/src/preload.c
index fb2149b..04ac6a0 100644
--- a/src/preload.c
+++ b/src/preload.c
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2011-2012 Intel Corporation.  All rights reserved.
+ * Copyright (c) 2011-2013 Intel Corporation.  All rights reserved.
  *
  * This software is available to you under a choice of one of two
  * licenses.  You may choose to be licensed under the terms of the GNU
